### ISSUE TITLE
Computing the softmax in run_squad.py

### ISSUE THREAD

**Issue by christopher5106** [google-research/bert#534]
You compute it over nbest results
1. Is  there a theoritical explanation of that ?
2. You sum start_logits and end_logits together without normalizing them with a softmax. Is not a risk of having one taking over the other ?
Thank you
